cmd/frontend/backend/orgs.go

```diff
@@ -3,6 +3,10 @@ package backend
 import (
 	"context"
 
+	"github.com/sourcegraph/sourcegraph/internal/database/dbutil"
+
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/envvar"
+
 	"github.com/cockroachdb/errors"
 
 	"github.com/sourcegraph/sourcegraph/internal/actor"
```

Comment:
```suggestion
	"github.com/cockroachdb/errors"

	"github.com/sourcegraph/sourcegraph/internal/database/dbutil"
	"github.com/sourcegraph/sourcegraph/cmd/frontend/envvar"
	"github.com/sourcegraph/sourcegraph/internal/actor"
```

note: the actual order of packages might still needs fix.